src/sage/categories/examples/finite_weyl_groups.py

```diff
@@ -157,7 +157,7 @@ def product(self, x, y):
         assert y in self
         return self(tuple(x.value[i] for i in y.value))
 
-    def degrees(self):
+    def degrees(self) -> tuple:
```

Comment:
A few more types for tuples (here) and lists (below) are missing. Could you please add them as well?